src/csv2docx/csv2docx.py

```diff
@@ -11,6 +12,7 @@ def create_output_folder(output_path: str) -> Path:
     Returns:
         A path to store output data.
     """
+
     path = Path(output_path)
     if not path.exists():
         path.mkdir(parents=True, exist_ok=True)
```

Comment:
If I change line 16 with:
```Python
path = Path(str(output_path).lstrip("/"))
```
then the condition that follows is never flagged regardless of input. I cannot explain. 

Thoughts?